This patch will be merge into the greeter but is here for testing.

--- chroot.orig/etc/gdm3/PostLogin/Default
+++ chroot/etc/gdm3/PostLogin/Default
@@ -73,9 +73,7 @@ if [ -r "${CAMOUFLAGE_SETTINGS}" ] ; then
     . "${CAMOUFLAGE_SETTINGS}"
     case "${TAILS_CAMOUFLAGE_OS}" in
 	win8)
-	    install -m 0644 \
-	        /usr/share/applications/tails-activate-win8-theme.desktop \
-	        /etc/xdg/autostart/
+        /usr/local/sbin/tails-prepare-win8-theme
 	    ;;
     esac
 fi

